### ISSUE TITLE
SA1009 is reported instead of SA1021

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#255]
In the following code, SA1009 is reported for the `)` token. In this situation, SA1009 should be suppressed and `SA1021` should be reported for the `-` token.

``` csharp
var x = (int) -3;
```
